dts/arm/st/wb/stm32wb55.dtsi

```diff
@@ -5,3 +5,11 @@
  */
 
 #include <st/wb/stm32wb.dtsi>
+
+/ {
+	soc {
+		adc1: adc@50040000 {
```

Comment:
Isn't it common to all WB SoC ?

Reply:
No, for example, it is on channel 12 for STM32WB15xx